src/helpers/TripHelper.js

```diff
@@ -40,6 +41,19 @@ class TripHelper {
       return Response.errorMessage(req, res, err.message, 500);
     }
   }
+
+  /**
+  * Should return city name of provided city id
+  * @static
+  * @param {object} tripTypeId city type ID
```

Comment:
Hey @victorkarangwa4  change this comment `city type ID` to something that is appropriate to the context.